docusaurus/docs/testing/extensions-test.md

```diff
@@ -0,0 +1,45 @@
+# Extensions Tests
+
+The Extensions framework is tested by the GitHub action `check-plugins.yaml`. This runs the script `./shell/scripts/test-plugins-build.sh`
+which runs the actual tests.
+
+The tests check the following:
+
+- Ensures that Verdaccio is installed and running locally (Verdaccio provides a local NPM registry that the tests can use to publish packages without having to publish to npm)
+- Verifies the build of the following, by building and then publishing them to the local registry
+  - Shell
+  - Yarn Creator packages
+  - Rancher components package
+- Verifies that a new app can be created with the `@rancher/app` creator and built (test case of a new extension in a new repository)
+- Verifies that an extension can be added with the `@rancher/pkg` creator to the new app and that the dashboard build passes with the new extension present
+- Verifies that the extension can be build standalone
+- Verifies that an extension can be added to the existing dashboard and that the dashboard build passes with the new extension present
+
+## Running Tests locally
+
+You can run the script `./shell/scripts/test-plugins-build.sh` locally in a terminal to run the test suite on a developer machine.
```

Comment:
When everything finally succeed, it creates automatically a commit for the tag:

![Screenshot 2023-04-21 at 10 52 44](https://user-images.githubusercontent.com/5009481/233592442-7498928b-dd19-4c40-9fb9-19ed0c3546a2.png)